src/databricks/labs/ucx/hive_metastore/migration_status.py

```diff
@@ -70,7 +70,7 @@ def get_seen_tables(self) -> dict[str, str]:
         seen_tables: dict[str, str] = {}
         for schema in self._iter_schemas():
             try:
-                tables = self._ws.tables.list(catalog_name=schema.catalog_name, schema_name=schema.name)
+                tables = list(self._ws.tables.list(catalog_name=schema.catalog_name, schema_name=schema.name))
```

Comment:
Why adding this list?

Reply:
because of `yield`. @nkvuong it's best to add comments :)

Reply:
it is only used once in the code below, so yield should be fine

Reply:
the SDK throws a NotFound exception if the schema does not exist - the try catch here won't work as the iterator is not consumed yet